components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/cache/AuthenticationContextCache.java

```diff
@@ -40,20 +41,26 @@ public class AuthenticationContextCache extends
     private static final Log log = LogFactory.getLog(AuthenticationContextCache.class);
     private static volatile AuthenticationContextCache instance;
     private boolean isTemporarySessionDataPersistEnabled = false;
+    private boolean isSessionDataStorageOptimizationEnabled = true;
```

Comment:
did we decide to have this set to true?

Reply:
Yes, if there is no config then optimization will happen,